python/py-jsonschema/Portfile

```diff
@@ -52,7 +52,8 @@ if {${name} ne ${subport}} {
                             port:py${python.version}-hatch-vcs \
                             port:py${python.version}-hatch-fancy-pypi-readme
 
-        if {${python.version} < 38} {
+        # On Yosemite and older OS-s rpds-py fails (requires rust)
```

Comment:
this should be indented properly with four spaces

Reply:
I don't see anything that's not indented to 4 spaces

Reply:
sorry, I missed that the first `if`-statement actually as removed.